Merge Conflict/Assets/Scripts/ConveyorBelt/ConveyorBeltMovement.cs

```diff
@@ -46,22 +96,22 @@ private void OnCollisionStay2D(Collision2D collision)
                     switch (MovingDirection)
                     {
                         case MovingDirection.DOWN:
-                            if (!componentHandler.IsOnConveyorBeltDiagonal)
+                            if (componentHandler.IsOnConveyorBeltDiagonal)
                             {
-                                var centerX = SizeOfPart.x / 2;
-                                var deltaX = centerX - collision.transform.position.x;
-
-                                translationToCenter = new Vector3(deltaX * Time.deltaTime * SpeedToCenter, 0, 0);
+                                translationToCenter = Vector3.zero;
                             }
                             else
                             {
-                                translationToCenter = Vector3.zero;
+                                var centerX = SizeOfPart.x / 2;
+                                var deltaX = centerX - collision.gameObject.GetComponent<RectTransform>().anchoredPosition.x;
```

Comment:
as I have commented before: use an variable to short the usage and to make it more readable